src/layer/layer_settings_manager.cpp

```diff
@@ -58,9 +58,19 @@ static std::string GetEnvironment(const char *variable) {
         result = GetAndroidProperty("debug.vulkan.screenshot");
     }
     return result;
+#elif defined(_WIN32)
```

Comment:
Do you know what compiler configuration cause these codes to generate warnings?

Reply:
This one is caused by removing `_CRT_SECURE_NO_WARNINGS`